pcap-linux.c

```diff
@@ -7136,6 +7151,28 @@ pcap_set_protocol_linux(pcap_t *p, int protocol)
 	return (0);
 }
 
+int
+pcap_set_fanout_linux(pcap_t *p, uint16_t mode, uint16_t group_id)
+{
+#ifdef PACKET_FANOUT
+	if (pcap_check_activated(p)) {
+		return (PCAP_ERROR_ACTIVATED);
+	}
+
+	if (mode >= 0) {
+		p->opt.fanout = (((int)mode) << 16) | ((int)group_id & 0xffff);
+	} else {
+		p->opt.fanout = -1;
+	}
+
+	return 0;
+#else
+	pcap_fmt_errmsg_for_errno(p->errbuf, PCAP_ERRBUF_SIZE,
+	    errno, "funout is not supported");
```

Comment:
Typeo: "funout".

Reply:
oops. fixed.

Reply:
It's funny that I made a typo pointing to a typo.